README.md

```diff
@@ -21,130 +22,66 @@ This allows you to build anything from simple websites to enterprise-grade intra
 
+
 ## Documentation
 
 You can find the latest documentation at [https://6.docs.plone.org/](https://6.docs.plone.org/volto/index.html).
 
 For links to trainings and videos, see [Other learning resources](https://6.docs.plone.org/volto/getting-started/others.html).
 
-## Supported Node.js versions
 
-Volto runs using [Node.js](https://nodejs.org/).
-We recommend using the current LTS version.
+## Supported Plone, Python, and Plone REST API versions
 
-- Node.js 20 LTS: Supported since Volto 17.
-- Node.js 18 LTS: Supported since Volto 17.
-- Node.js 16: No longer supported. It was supported from Volto 14 - 16.
-- Node.js 14: No longer supported. It was supported from Volto 8.8.0 - 16.
-- Node.js 12: No longer supported. It was supported from Volto 4 - 15.
-- Node.js 10: No longer supported. It was supported from Volto 1 - 12.
+See [Plone, Python, and Plone REST API compatibility](https://6.docs.plone.org/volto/contributing/version-policy.html#version-policy-plone-python-and-plone-rest-api-compatibility)
 
-## Supported Plone and Python versions
+See the [Plone Release Schedule](https://plone.org/download/release-schedule) for details of maintenance and support.
 
-Volto is the default UI for Plone 6.
-It will work for all released Plone 6 versions.
 
-For the Plone 5 series, the [latest released version of Plone 5](https://plone.org/download/releases) is recommended.
+## Supported Node.js versions
 
-The versions of Python that are supported in Volto depend on the version of Plone that you use.
+See [Node.js version policy](https://6.docs.plone.org/volto/contributing/version-policy.html#version-policy-plone-python-and-plone-rest-api-compatibility).
 
-| Plone | Python       | Volto        |
-| ----- | ------------ | ------------ |
-| 6.0   | 3.8-3.11     | 16.0 or 17.0 |
-| 5.2   | 2.7, 3.6-3.8 | 15.0         |
 
 ## Supported browsers
 
-Volto works well with the current version of any modern browser, including their mobile flavors: Chrome, Firefox, Safari, and Edge.
+See [Supported browsers](https://6.docs.plone.org/volto/contributing/version-policy.html#version-policy-supported-browsers).
 
-We do not guarantee that outdated browsers such as Internet Explorer 11 are supported by Volto.
 
-## Known good set of versions for backend packages
-
-On Plone 6, we recommend using the known good set (KGS) of package versions that are specified in the Plone release.
-
-On Plone 5, Volto is currently tested with the following packages pinned to specific versions, and we recommend using the same versions, which are:
+## Volto in Production
 
--   `plone.restapi` 9.2.0
--   `plone.rest` 3.0.1
--   `plone.volto` 4.1.0
+Volto is actively developed since 2017.
+It has been used in production since 2018 on numerous websites.
+See [Awesome Volto](https://github.com/collective/awesome-volto#websites-built-with-volto) to add your website to the growing list.
```

Comment:
@stevepiercy I think it is important to keep this list in the README for the time being. It is still important that people see that Volto is used a lot. This list should go on plone.org in the future and we should have success stories for each of those. Moving this to a collective project isn't the right move IMHO.

Reply:
@tisto this PR is a good first step toward unifying lists of Volto websites and other Awesome tidbits.

If a list of Volto websites should go on plone.org, why not do it now? Now there is a single list of websites with validated links! There's a Plone Tune Up day on Fri, Dec 15. 😁 

I disagree that multiple lists of Volto websites is a good thing. Separate lists means none are authoritative and they dilute the marketing effectiveness of one another.

They are burdensome to maintain in multiple locations. No one maintained the links in this README, as demonstrated in the work required to update the outdated links in https://github.com/collective/awesome-volto/pull/17. At least collective has CI to check links on each PR.

Reply:
@tisto it's been a few days since I posted my explanation for removing the list, so I assume that you have no objection to its removal. I'll wait a few days to give you a chance to respond before merging.

Reply:
@stevepiercy I feel the same about @tisto opinion. Until we don't have a central stage place for them, we should keep them here. In fact, this is something that should go directly into @plone/marketing-communications team radar, not ours.

Reply:
@sneridagh we do have a [central storage place for them](https://github.com/collective/awesome-volto?tab=readme-ov-file#websites-built-with-volto).

If y'all insist that the list of links to websites using Volto remains in the README, then I insist that there be a linkchecker on this README. Broken links make us look bad, and it was very bad. It might not be a bad thing to add the [linkchecker from awesome-volto](https://github.com/collective/awesome-volto/blob/master/.github/workflows/main.yaml) to do that anyway, as well as for all READMEs, since they have no checking whatsoever.

Also if the list of links remains, then I insist that they be the last thing in the README. I have no data to back up my assumption, but I think that when the Contributing section is buried after a wall of links, then newcomers are more likely to not see it, which in turn creates a lot of extra work.

To move this PR forward, first y'all need to decide whether the list of links should remain or be removed, and if removed then under what conditions.

### Conditions to remove list of links

1. Links are centralized at [`awesome-volto`](https://github.com/collective/awesome-volto?tab=readme-ov-file#websites-built-with-volto). This is done.
2. Links are published on plone.org, according to @plone/marketing-communications'a direction.
3. Your suggestions here.

### Next steps, depending upon above decision

1. Now merge PR as is without links.
2. Wait until @plone/marketing-communications decides what to do with the list of links.
3. Now copy the validated list of links from Awesome Volto as the final item in the README.

No matter what, I would add linkchecking to all the READMEs as a separate PR.

Please let me know. Thank you!

Reply:
@sneridagh and I chatted to retain the links to sites, and add the CI link checker, and to do option 3 in Next Steps.

Reply:
During today's Plone Tune-up, @plone/marketing-communications added a new page at [They use Plone 6](https://plone.org/why-plone/they-use-plone/they-use-plone-6) in https://github.com/plone/plone.org/issues/154, satisfying item 2 under the "Conditions to remove list of links".

We now have three places to keep in sync with each other. @sneridagh and @tisto are you sure you still want me to restore the links here on the README and implement that automatic checking? I assume the answer is "yes", unless and until you both say otherwise.

Going forward together with the Marketing Team (@rioksane @skleinfeldt), we should treat Awesome Volto as the authoritative source. Here's my suggestion, and I'll update this PR accordingly.

---

## Volto in production

Volto has been actively developed since 2017.
It has been used in production since 2018 on numerous websites.

The authoritative source of the list of Volto websites in production is maintained at [Awesome Volto](https://github.com/collective/awesome-volto#websites-built-with-volto).

The Plone Marketing Team copy-pastes its content on a quarterly basis into [They use Plone 6](https://plone.org/why-plone/they-use-plone/they-use-plone-6).

To ensure your website gets the greatest exposure, add it both to [Awesome Volto](https://github.com/collective/awesome-volto#websites-built-with-volto) and this README.

[Monstrously huge list of links that needs to be kept in synch.]

---

Reply:
Done in [bad7b67](https://github.com/plone/volto/pull/5511/commits/bad7b67cca090e121fd29a856d227e9a04561ad9)

Reply:
Added awesome_bot to GitHub workflows. Let's see how it does.

https://github.com/plone/volto/actions/runs/7596509389/job/20690300977?pr=5511